### PR TITLE
0.14.1

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

**Comment by domenukk** [comment]
cc @andreafioraldi

**Comment by domenukk** [comment]
We should update ctor to 0.2.9 to fix ci

**Comment by domenukk** [comment]
> We should update ctor to 0.2.9 to fix ci

done in #2713

File: Cargo.toml [hunk 1]
```diff
@@ -49,7 +49,7 @@ exclude = [
 ]
 
 [workspace.package]
-version = "0.14.0"
+version = "0.14.1"
 license = "MIT OR Apache-2.0"
 
 [workspace.dependencies]
```
Comment (Marcondiro) [inline Cargo.toml:55]: In case you missed them, also the internal deps down here will need to be dumped

Comment (domenukk) [reply Cargo.toml:55]: There was a script for updating these I think? Maybe the script needs a fix then? cc @tokatoka

Comment (tokatoka) [reply Cargo.toml:55]: idk what i did is is open vscode and ctrl f and replace

no i'll just fix manually..

**Pull request merged**